app/src/main/java/com/example/messengerAndroid/ui/myContacts/MyContactsActivity.kt

```diff
@@ -14,15 +14,17 @@ import com.example.messengerAndroid.R
 import com.example.messengerAndroid.databinding.ActivityMyContactsBinding
 import com.example.messengerAndroid.databinding.DialogAddContactBinding
 import com.example.messengerAndroid.data.contactsRepository.contactModel.User
-import com.example.messengerAndroid.ui.contactsViewModel.ContactsViewModel
-import com.example.messengerAndroid.ui.contactsViewModel.SwipeToDeleteCallback
-import com.example.messengerAndroid.ui.contactsViewModel.getUserPosition
+import com.example.messengerAndroid.ui.myContacts.adapter.ContactsAdapter
+import com.example.messengerAndroid.ui.myContacts.adapter.UserActionListener
+import com.example.messengerAndroid.ui.myContacts.contactsViewModel.ContactsViewModel
+import com.example.messengerAndroid.ui.myContacts.contactsViewModel.SwipeToDeleteCallback
 import com.google.android.material.snackbar.Snackbar
 
 
 class MyContactsActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityMyContactsBinding
+    private val viewModel: ContactsViewModel by viewModels()
```

Comment:
It may  sound like nonsense but I prefer sorting my variable by their type and access modifiers (from override to private), e.g.:

private val a = 0

private var b = 1

private lateinit var user: User

Reply:
https://kotlinlang.org/docs/coding-conventions.html#modifiers-order